internal/presenters/presenter_sarif_results_pretty.go

```diff
@@ -55,129 +115,88 @@ func convertSarifToFindingsList(input sarif.SarifDocument) []Finding {
 				location = result.Locations[0]
 			}
 
-			findings = append(findings, Finding{
-				ID:       result.RuleID,
-				Severity: severity,
-				Title:    title,
-				Path:     location.PhysicalLocation.ArtifactLocation.URI,
-				Line:     location.PhysicalLocation.Region.StartLine,
-				Message:  result.Message.Text,
-			})
-		}
-	}
-	return findings
-}
+			isIgnored := len(result.Suppressions) > 0
 
-type TestMeta struct {
-	OrgName  string
-	TestPath string
-}
+			var ignoreProperties []FindingProperty
 
-func PresenterSarifResultsPretty(input sarif.SarifDocument, meta TestMeta) (string, error) {
-	findings := convertSarifToFindingsList(input)
-	summary := sarif_utils.CreateCodeSummary(&input)
-
-	str := fmt.Sprintf(`
-Testing %s ...
-%s
-%s
-%s
-`,
-		meta.TestPath,
-		renderFindings(SortFindings(findings, summary.SeverityOrderAsc)),
-		renderSummary(summary, meta),
-		getTip(),
-	)
+			ignoreProperties = append(ignoreProperties, FindingProperty{
+				Label: "Path",
+				Value: fmt.Sprintf("%s, line %d",
+					location.PhysicalLocation.ArtifactLocation.URI,
+					location.PhysicalLocation.Region.StartLine,
+				),
+			})
 
-	return str, nil
-}
+			ignoreProperties = append(ignoreProperties, FindingProperty{
+				Label: "Info",
+				Value: result.Message.Text,
+			})
 
-func renderFindings(findings []Finding) string {
-	if len(findings) == 0 {
-		return ""
-	}
+			ignoreProperties = append(ignoreProperties, FindingProperty{
+				Label: "",
+				Value: "",
+			})
 
-	response := "\nOpen Issues\n\n"
+			for _, suppression := range result.Suppressions {
+				expiration := ""
+				if suppression.Properties.Expiration != nil {
+					expiration = *suppression.Properties.Expiration
+				}
 
-	for _, finding := range findings {
-		response += fmt.Sprintf(` %s %s
-   Path: %s, line %d
-   Info: %s
+				ignoreProperties = append(ignoreProperties, FindingProperty{
+					Label: "Expiration",
+					Value: expiration,
+				})
 
-`, renderInSeverityColor(finding.Severity, fmt.Sprintf("✗ [%s]", strings.ToUpper(finding.Severity))), renderBold(finding.Title), finding.Path, finding.Line, finding.Message)
-	}
+				ignoreProperties = append(ignoreProperties, FindingProperty{
+					Label: "Category",
+					Value: strings.Replace(string(suppression.Properties.Category), "wont-fix", "Won't fix", 1),
+				})
 
-	return response
-}
+				// TODO: Verify date formatting
+				s, err := time.Parse(time.RFC3339, suppression.Properties.IgnoredOn)
 
-func getTip() string {
-	return `
-💡 Tip
+				if err == nil {
```

Comment:
Suggestion: How about printing the original timestamp when we experience a parser error